lib/Doctrine/DBAL/Connection.php

```diff
@@ -999,9 +999,9 @@ public function fetchAllAssociative(string $query, array $params = [], array $ty
      * Prepares and executes an SQL query and returns the result as an associative array with the keys
      * mapped to the first column and the values mapped to the second column.
      *
-     * @param string                                           $query  SQL query
-     * @param array<int, mixed>|array<string, mixed>           $params Query parameters
-     * @param array<int, int|string>|array<string, int|string> $types  Parameter types
+     * @param string                                                               $query  SQL query
+     * @param array<int, mixed>|array<string, mixed>                               $params Query parameters
+     * @param array<int, int|string|Type|null>|array<string, int|string|Type|null> $types  Parameter types
```

Comment:
Static analysis failed, because query builder may use `Type` objects passed to these Connection methods.